lgc/builder/llpcBuilderImplSubgroup.cpp

```diff
@@ -191,7 +191,7 @@ Value* BuilderImplSubgroup::CreateSubgroupBallot(
     pBallot = CreateBitCast(pBallot, VectorType::get(getInt32Ty(), 2));
 
     ElementCount elementCount = pBallot->getType()->getVectorElementCount();
-    return CreateShuffleVector(pBallot, ConstantVector::getSplat(elementCount, getInt32(0)), { 0, 1, 2, 3 });
+    return CreateShuffleVector(pBallot, ConstantVector::getSplat(elementCount, getInt32(0)), ArrayRef<uint32_t>{ 0, 1, 2, 3 });
```

Comment:
Pay attention to the line limit: 120 character. More characters are added by "ArrayRef<uint32_t>" :) Please check all modifications.